### DISCUSSION TITLE
Phospho site

### DISCUSSION THREAD

**Discussion by ABRai11** [vdemichev/DiaNN#814] [Q&A] [answered]
Hello,

I am new to DIA-NN. I had searched a phospho enriched DIA data in DIA-NN and found modification as Unimod: 21. Is it possible to get the phospho site as SXXX/TXX (S200/T23)?

Thank you.

**Comment by vdemichev** [comment, marked as answer]
Only using some R package for this.

**Comment by ABRai11** [comment]
Could you please give the link/R package?

**Reply by vdemichev** [reply]
https://github.com/MannLabs/alphamap

**Reply by vdemichev** [reply]
For example. But I guess there are some other packages which can help too

**Comment by ABRai11** [comment]
Ok. Thank you so much. I will check this.

**Comment by ABRai11** [comment]
The Alpha map that you suggested is to visualize the protein modifications. Is there any other option to get the site information for a list of modified proteins?
